articles/tech-resources/changes-to-sparkpost-enrichment-period-open-click-events.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Changes to SparkPost’s “Enrichment” period for open and click events"
+description: "notification of changes to enrichment period for all SPC users"
```

Comment:
SPC is an internal term

Reply:
Updated it with Chris's suggestion. Good catch, thank you!